### PR TITLE
[DOCS] Update the mentoring projects

### PR DESCRIPTION
**Pull request by TonyBagnall**
have done some work on the project pages, it would be good if someone else could proof read it and add anything they think would be good.

### CODE REVIEW

File: docs/mentoring.md [hunk 1]
```diff
@@ -3,10 +3,11 @@
 
 # Mentoring and Projects
 
-`aeon` runs a range of short projects interacting with the community and the code
+`aeon` runs a range of short to medium duration projects interacting with the community
+and the code
 base. These projects are designed for internships, usage as part of
 undergraduate/postgraduate projects at academic institutions, and as options for
-programs such as [Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/).
+programs such as [Google Summer of Code (GSoC)](https://github.com/aeon-toolkit/aeon-admin/blob/main/gsoc/gsoc-2024-projects.md).
```
Comment (MatthewMiddlehurst) [inline docs/mentoring.md:10]: I would keep this general, GSoC already has its advertisements (which we have to remove now also)

**Review by MatthewMiddlehurst — COMMENTED** [review]
I would rather remove the ones which are already taken IMO.

**Comment by TonyBagnall** [comment]
> I would rather remove the ones which are already taken IMO.

Which ones are any taken? I was going to have a completed list once we have any, but Im not aware of any that have been assigned yet

**Comment by MatthewMiddlehurst** [comment]
Fair enough, getting ahead of myself a bit there 🙂.

**Review by chrisholder — APPROVED** [review]

**Pull request merged**